test/test_sparse.py

```diff
@@ -292,17 +274,17 @@ def fn(x):
             x.requires_grad_(True)
             gradcheck(fn, (x,), check_sparse_nnz=True)
 
-        i = self.index_tensor([
-            [0, 1, 2, 2],
-            [0, 0, 0, 3],
-            [0, 0, 1, 4],
-        ])
-        # we don't have to_dense for half types on CPU because it is implemented
-        # with a slower add_ operation
-        for dtype in [torch.float16, torch.float32, torch.float64] if self.device != 'cpu' else [torch.float32, torch.float64]:
-            v = self.value_tensor([2, 1, 3, 4]).to(dtype=dtype)
-            x = self.sparse_tensor(i, v, torch.Size([3, 4, 5]))
-            res = self.value_tensor([
+        for value_type in [torch.double]:
```

Comment:
Shouldn't this just set value_type = dtype?

Reply:
Sure, my mistake